### PR TITLE
Coffee match component

### PR DESCRIPTION
**Pull request by DEJENERUTA**

### CODE REVIEW

File: src/CoffeeMatch.js [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+      name: "zala",
+      image: "./assets/zala.png",
+      rating: 87,
+    },
+    {
+      name: "zenu",
+      image: "./assets/zenu.png",
+      rating: 82,
+    },
+  ];
```
Comment (bence-toth) [inline src/CoffeeMatch.js:60]: Not sure what this is doing here. Should this be a prop?

File: src/CoffeeMatch.js [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+  return (
+    <div>
+      <h3>CoffeeMatch</h3>
+      <div className="coffee-list">
```
Comment (bence-toth) [inline src/CoffeeMatch.js:64]: Unused classname?

File: src/CoffeeMatch.js [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+        {items.map((item) => (
+          <div className="coffee-item">
```
Comment (bence-toth) [inline src/CoffeeMatch.js:66]: Unused classname? Also missing `key`?

**Review by bence-toth — CHANGES_REQUESTED** [review]

File: src/CoffeeMatch.js (hunk 1 above)
Comment (DEJENERUTA) [reply src/CoffeeMatch.js:60]: I moved the above code to app.js and I used name, image and qr as a props in coffeeMatch.js

File: src/App.js [hunk 4]
```diff
@@ -1,9 +1,78 @@
...
+        <div>
+          {items.map((item) => (
+            <div>
+              <img src={item.image} alt={item.name} />
+              <h2>{item.name}</h2>
+              <h3>{item.qr}</h3>
+            </div>
+          ))}
+        </div>
+      </div>
```
Comment (bence-toth) [inline src/App.js:75]: Here you probably want to do something like this:

```suggestion
      <CoffeeMatch matches={matches} />
```

File: src/App.js [hunk 5]
```diff
@@ -1,9 +1,78 @@
 import "./App.css";
 
 const App = () => {
+  [
```
Comment (bence-toth) [inline src/App.js:4]: Assign this to a const:

```suggestion
  const matches = [
```

File: src/CoffeeMatch.js [hunk 6]
```diff
@@ -0,0 +1,27 @@
+import React from "react";
+
+const CoffeeMatch = () => {
+  [
+    {
+      name,
+      image,
+      qr,
+    },
+  ];
```
Comment (bence-toth) [inline src/CoffeeMatch.js:10]: Not sure why this is here

File: src/CoffeeMatch.js [hunk 7]
```diff
@@ -0,0 +1,27 @@
+import React from "react";
+
+const CoffeeMatch = () => {
```
Comment (bence-toth) [inline src/CoffeeMatch.js:3]: You'll need a prop here:

```suggestion
const CoffeeMatch = ({matches}) => {
```

File: src/CoffeeMatch.js [hunk 8]
```diff
@@ -0,0 +1,27 @@
...
+      name,
+      image,
+      qr,
+    },
+  ];
+  return (
+    <div>
+      <h3>CoffeeMatch</h3>
+      <div>
+        {items.map((item) => (
```
Comment (bence-toth) [inline src/CoffeeMatch.js:15]: Let's iterate through the `matches` prop here:

```suggestion
        {matches.map((item) => (
```

**Review by bence-toth — CHANGES_REQUESTED** [review]

File: src/CoffeeMatch.js [hunk 9]
```diff
@@ -0,0 +1,27 @@
...
+          <div>
```
Comment (bence-toth) [inline src/CoffeeMatch.js:16]: Missing key here:
```suggestion
          <div key={item.name}>
```

**Review by bence-toth — CHANGES_REQUESTED** [review]

File: src/App.js [hunk 10]
```diff
@@ -1,9 +1,67 @@
-import "./App.css";
+import CoffeeMatch from "./CoffeeMatch";
 
 const App = () => {
+  const matches = [
+    {
+      name: "aynu",
+      image: "./assets/aynu.png",
```
Comment (bence-toth) [inline src/App.js:7]: Let's use the full URLs from the website, let's not upload images to the assets folder here.

**Review by bence-toth — CHANGES_REQUESTED** [review]

File: src/App.js [hunk 11]
```diff
@@ -1,9 +1,67 @@
...
+      qr: 88,
```
Comment (bence-toth) [inline src/App.js:8]:
```suggestion
      rating: 88,
```

File: src/CoffeeMatch.js [hunk 12]
```diff
@@ -0,0 +1,20 @@
...
+const CoffeeMatch = ({ matches }) => {
+  return (
+    <div>
+      <h3>Here is your Coffee match!</h3>
+      <div>
+        {matches.map((item) => (
+          <div key={item.name}>
+            <img src={item.image} />
+            <h2>{item.name}</h2>
+            <h3>{item.qr}</h3>
```
Comment (bence-toth) [inline src/CoffeeMatch.js:12]:
```suggestion
            <h3>{item.rating}</h3>
```

**Review by bence-toth — CHANGES_REQUESTED** [review]

File: src/CoffeeMatch.js [hunk 13]
```diff
@@ -0,0 +1,23 @@
+//how to get image from www.impactroasters.dk/coffee/ and add it to the array of matches
```
Comment (bence-toth) [inline src/CoffeeMatch.js:1]: I can see that it's jpg files on the website, so maybe let's have the png files in this repo after all. I'll comment on the other file how we should do that.

File: src/App.js [hunk 14]
```diff
@@ -1,9 +1,68 @@
+import CoffeeMatch from "./CoffeeMatch";
+import "./assets/aynu.png";
```
Comment (bence-toth) [inline src/App.js:2]: Import the image like this:
```suggestion
import AynuImage from "./assets/aynu.png";
```

File: src/App.js [hunk 15]
```diff
@@ -1,9 +1,68 @@
...
 import "./App.css";
-
 const App = () => {
+  const matches = [
+    {
+      name: "aynu",
+      image: "./assets/aynu.png",
```
Comment (bence-toth) [inline src/App.js:8]: Use the imported image like this:

```suggestion
      image: AynuImage,
```

File: src/App.css [hunk 16]
```diff
@@ -1 +1,43 @@
...
+  margin: 0 auto;
+  width: 100%;
+  height: 100%;
+  text-transform: capitalize;
+}
+img {
+  width: 150px;
+  height: 150px;
+  border-radius: 25px;
+}
```
Comment (bence-toth) [inline src/App.css:43]: Let's move this to another CSS file called `CoffeeMatch.css` and import it in `CoffeeMatch.js`

File: src/App.css [hunk 17]
```diff
@@ -1 +1,43 @@
...
+  gap: 2em;
+  border: 1px solid #000;
+  border-radius: 5px;
+  padding: 10px;
+  margin: 0 auto;
+  width: 100%;
+  height: 100%;
+  text-transform: capitalize;
+}
+img {
```
Comment (bence-toth) [inline src/App.css:39]: Let's be more specific here, use a class name. Generally don't use element selectors like `img` or `h1`.

File: src/App.css [hunk 18]
```diff
@@ -1 +1,43 @@
...
+  grid-gap: 1rem;
+  margin: 0 auto;
+  justify-content: center;
+  align-items: center;
+  width: 40%;
+  height: 40%;
+  background-color: #f5f5f5;
+}
+h1,
+h3 {
```
Comment (bence-toth) [inline src/App.css:19]: Let's be more specific here, use class names. Generally don't use element selectors like `img` or `h1`.

**Review by bence-toth — CHANGES_REQUESTED** [review]

File: src/CoffeeMatch.css [hunk 19]
```diff
@@ -0,0 +1,43 @@
+.app {
+  margin: 0 auto;
+  width: 100%;
+  text-align: center;
+}
```
Comment (bence-toth) [inline src/CoffeeMatch.css:5]: This part should go back to the `App.css` file

Comment (bence-toth) [reply src/CoffeeMatch.css:5]: Can you please remove it from here?

**Review by bence-toth — CHANGES_REQUESTED** [review]

**Review by bence-toth — APPROVED** [review]

**Pull request merged**